### ISSUE TITLE
export: make transformation optional for actions that export users to apps

### ISSUE THREAD

**Issue by zapateo** [krenalis/krenalis#933]
Since the export matching property is automatically transformed (in case the export mode is create or create+update), it might be reasonable to not want to export any other properties and just export that property.

Currently, however, this is not possible because the export matching property cannot be explicitly transformed (Krenalis implicitly transforms it), and the transformation is mandatory, thus forcing the user to export at least one other property.

* Is the use case indicated above reasonable? Or does it never happen in practice?
* If it is reasonable, then does it make sense to make the transformation function optional instead of mandatory?

**Comment by gazerro** [comment]
> Is the use case indicated above reasonable? Or does it never happen in practice?

Yes, this should only be possible in `Create` mode. It makes no sense to choose `CreateOrUpdate` mode without a transformation since no update could be performed.

> If it is reasonable, then does it make sense to make the transformation function optional instead of mandatory?

It should be handled like the case of sending events to apps where there are no required properties. If you mean allowing the user to explicitly choose whether to have a transformation or not, then no. The use case is so uncommon that it doesn't warrant special treatment beyond a mapping without the obligation to map any property. However, if a function is chosen, it remains mandatory to select at least one output property.
